lib/codit/population/networks/city.py

```diff
@@ -104,6 +120,9 @@ def build_households(people):
         for j, age in enumerate(ages):
             indiv = people[j + assigned]
             indiv.age = age
+            # Initialise Home instance with (coordinates and building_type) to each person's home attribute within the population
+            indiv.home = Home(home[0], home[1], home[2])
```

Comment:
could you write (which is maybe clearer):

`indiv.home = Home(*home)`

Reply:
Yes, nice way!